File: Sesion 6/Marco teorico/10QUÉ PROPIEDADES TIENEN LOS MIEMBROS STATIC/10.java
class Ejemplo {
    static int contador = 0;

    Ejemplo() {
        contador++;
    }

    static void mostrarContador() {
        System.out.println("El contador es: " + contador);
    }
}

public class Main {
    public static void main(String[] args) {
        Ejemplo e1 = new Ejemplo();
        Ejemplo e2 = new Ejemplo();
        Ejemplo e3 = new Ejemplo();

        // Llamada al método estático desde la clase (no se necesita una instancia)
        Ejemplo.mostrarContador();
    }
}

File: Sesion 6/Marco teorico/10QUÉ PROPIEDADES TIENEN LOS MIEMBROS STATIC/10.py
class Ejemplo:
    contador = 0

    def __init__(self):
        Ejemplo.contador += 1

    @staticmethod
    def mostrar_contador():
        print("El contador es:", Ejemplo.contador)

e1 = Ejemplo()
e2 = Ejemplo()
e3 = Ejemplo()
Ejemplo.mostrar_contador()

